pyperformance/data-files/benchmarks/bm_pypdf/run_benchmark.py

```diff
@@ -0,0 +1,89 @@
+"""
+A simple pypdf benchmark.
+
+Adapted from pypdf's own benchmarks:
+
+   https://github.com/py-pdf/pypdf/blob/main/tests/bench.py
```

Comment:
Assuming it makes sense to do so, could we contribute a change upstream to make their benchmarks compatible with pyperformance (and pull them in like we do for the pyston benchmarks), instead of landing a new benchmark here?